shared/lib/delegation/caveatBuilder/exactExecutionBuilder.ts

```diff
@@ -0,0 +1,27 @@
+import { encodePacked } from '@metamask/abi-utils';
+import { Hex, bytesToHex } from '@metamask/utils';
+import { Caveat } from '../caveat';
+import { DeleGatorEnvironment } from '../environment';
+
+export const exactExecution = 'exactExecution';
+
+export function exactExecutionBuilder(
+  environment: DeleGatorEnvironment,
+  expectedExecution: Hex,
```

Comment:
The intent of the builders is to give you a typed interface to provide the terms separately.

Meaning we'd want arguments for `to`, `value` and `data`.